package exception;

public class E001_Demo {
			
	public static void main(String[] args) {
		
		System.out.println("Program started");
		
		try {
			
			int a = 10;
			int j = a/2;
			System.out.println(j);
			
		}
		catch (ArithmeticException e) 
		{
				e.printStackTrace();
		}
		
		System.out.println("Program ended");
		
	}
}
